.github/workflows/release.yml

```diff
@@ -38,7 +38,7 @@ jobs:
         # https://github.com/actions/setup-python
         uses: actions/setup-python@v3
         with:
-          python-version: 3.9
+          python-version: 3.12
```

Comment:
No, we should keep the oldest possible version here. See comment line 24.

Reply:
Reverted.